qcodes/instrument_drivers/QuTech/S5i.py

```diff
@@ -14,49 +14,67 @@ class S5i(Instrument):
 
     Args:
         name (str): name of the instrument.
-
         spi_rack (SPI_rack): instance of the SPI_rack class as defined in
             the spirack package. This class manages communication with the
             individual modules.
-
         module (int): module number as set on the hardware.
     """
 
-    def __init__(self, name, spi_rack, module, **kwargs):
+    def __init__(self, name, spi_rack, module, frequency=41e6, enable_output=1,
+                 output_level=0, **kwargs):
```

Comment:
`enable_output` - why does it have an integer value? is it required by `S5i_module`? 

my suggestion would be to change the type to boolean, so that `enable_output=True` is passed to `__init__`, and then, if `S5i_module` requires that, pass `enable_output=int(enable_output)` to `S5i_module` constructor. As a bonus, this way the initial value for the "enable_output" parameter below will be set correctly.

Reply:
done